waku/node/config.nim

```diff
@@ -36,7 +36,7 @@ type NetConfig* = object
 type NetConfigResult* = Result[NetConfig, string]
 
 
-template ip4TcpEndPoint(address, port): MultiAddress =
+template ip4TcpEndPoint*(address, port): MultiAddress =
```

Comment:
Isn't there a way to make the tests without changing the scope of that?
If possible, I recommend only testing the public procs and don't change the scope of private ones. If you still need to use these private procs you may need to `include` the `../../waku/node/config,` in the test, instead of `import`.

Reply:
Makes sense! I changed it to `import` instead. I noticed that for some reason really few test files use `import` - I wonder how many private procs have been turned into public for testing, probably a lot. Anyway, I agree on the principle of making the least amount possible of changes in prod code that aren't related to prod logic.